src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/OrchestrationConstants.java

```diff
@@ -6,6 +6,8 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class OrchestrationConstants {
 
+    public static final String LINE_SEPARATOR = System.lineSeparator();
```

Comment:
Should it really be `System.lineSeparator()` ? It's going to make the output dependent on the platform the app is running on. Like if they migrate one day from Windows to Linux or the other way round it can change what template renders.

Reply:
Do you think it should just be hard-coded then to '\n' or what would you advise?

Reply:
Good point. But I think any other way will be even more deterministic/arbitrary.